dbms/src/Storages/DeltaMerge/ReadThread/SegmentReadTaskScheduler.h

```diff
@@ -86,12 +88,14 @@ class SegmentReadTaskScheduler
 
     MergedTaskPool merged_task_pool;
 
-    std::atomic<bool> stop;
+    std::atomic<bool> stop{false};
     std::thread sched_thread;
 
     LoggerPtr log;
 
     // To count how many threads are waitting to add tasks.
     std::atomic<Int64> add_waittings{0};
+
+    bool enable_data_sharing{true};
```

Comment:
Move it next to the `std::atomic<bool> stop` can share the align and reduce memory consumption of this instance. (Though we will only create one instance for this scheduler)